src/frontend/js/components/PaymentButton/hooks/useTerms.tsx

```diff
@@ -66,12 +53,14 @@ export const useTerms = ({
             label={
               <>
                 {intl.formatMessage(messages.termsMessage)}{' '}
-                <button
-                  onClick={openContract}
+                <a
+                  href={context.site_urls.terms_and_conditions ?? '#'}
```

Comment:
In which cases this attribute could not be defined?

Reply:
If the page does not exist.